README.md

```diff
@@ -1,7 +1,190 @@
-### flux-multi-cluster-utilities
-Repository for Fractale's scheduling thrust artifacts.
+## Flux Multi-Cluster Utilities
+This repository houses the artifacts developed as part of the 
+FRACTALE SI's Center-Level Scheduling Thrust. 
+One of the goals of this thrust is to enable scheduling jobs across different 
+clusters of a site, with the objective of 
+improving job performance, turnaround times, system throughput and utilization. 
+A future goal is to enable such scheduling across both HPC clusters and on-premises 
+Cloud setups.
 
-### License
+To this effect, the first building block is a jobtap plugin 
+that allows a job to be delegated (submitted) from the 
+current flux instance to a different flux instance (given its URI).
+
+### Build Instructions
+
+We use GNU Autotools to build this plugin as follows.
+
+```
+./autogen.sh
+./configure --prefix=<install-path>
+make && make install
+```
+The `delegate.so` plugin will be installed in `<install-path>/lib/flux/job-manager/plugins/`.
+
+If no `<install-path>` is specified, it is installed in `flux-multi-cluster-utilities/src/job-manager/plugins/.libs`.
```

Comment:
I think actually what's going on is `make` puts it in `src/job-manager/plugins/.libs` and `make install` puts it in `<install-path>/lib/flux/job-manager/plugins/`. The way this sentence above describes it makes it sound like it's either/or